frontend/package.json

```diff
@@ -3,28 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.2.28",
-    "@fortawesome/free-regular-svg-icons": "^5.13.0",
-    "@fortawesome/free-solid-svg-icons": "^5.13.0",
-    "@fortawesome/react-fontawesome": "^0.1.9",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.5.0",
-    "@testing-library/user-event": "^7.2.1",
-    "babel-polyfill": "^6.26.0",
-    "cross-fetch": "^3.0.4",
-    "date-fns": "^2.13.0",
-    "node-sass": "^4.14.1",
-    "prop-types": "^15.7.2",
-    "react": "^16.13.1",
-    "react-datepicker": "^2.14.1",
-    "react-dom": "^16.13.1",
-    "react-redux": "^7.2.0",
-    "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.1",
-    "react-select": "^3.1.0",
-    "redux": "^4.0.5",
-    "redux-thunk": "^2.3.0",
-    "typescript": "^3.8.3"
+    "@fortawesome/fontawesome-svg-core": "1.2.36",
+    "@fortawesome/free-regular-svg-icons": "5.15.4",
+    "@fortawesome/free-solid-svg-icons": "5.15.4",
+    "@fortawesome/react-fontawesome": "0.1.15",
+    "date-fns": "2.23.0",
+    "react": "17.0.2",
+    "react-datepicker": "4.2.1",
+    "react-dom": "17.0.2",
+    "react-redux": "7.2.5",
+    "react-router-dom": "5.3.0",
+    "react-select": "4.3.1",
+    "redux": "4.1.1",
+    "redux-thunk": "2.3.0"
+  },
+  "devDependencies": {
+    "react-scripts": "4.0.3",
```

Comment:
With our build deps as `devDependencies` we can run `npm audit --production` to get the actual number of affected vulnerabilities not thousands of nonse Regex DoS CVEs that only affect our build tools. This is official advice from Create React App too (https://github.com/facebook/create-react-app/issues/11174)